code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -103,11 +106,14 @@
 	max_shells = 20
 	burst_count = 4
 	caliber = list("12mm" = 1)
-	ammo_type = "/obj/item/ammo_casing/a12mm"
+	ammo_type = "/obj/item/ammo_casing/a12mm/assault"
 	mag_type = "/obj/item/ammo_storage/magazine/a12mm"
 	fire_sound = 'sound/weapons/Gunshot_c20.ogg'
 	load_method = 2
 
+/obj/item/weapon/gun/projectile/automatic/xcom/lockbox
```

Comment:
This should not be here - you've inserted it in the middle of another type, and this position would actually mess up the gun bitflags.